src/components/dropdown-menu/DropdownMenu.tsx

```diff
@@ -155,7 +155,7 @@ export const DropdownMenu = {
   Root: DropdownMenuRoot,
   Separator: DropdownMenuSeparator,
   SubMenu: DropdownMenuSubMenu,
-};
+});
```

Comment:
[Question] Are we keeping `Root` for backwards compatibility?

Reply:
Yeah that's what I had in mind but actually I'll remove it, it's an easy to fix breaking change

Reply:
I went ahead and removed it so I could merge.